package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "amcharts3-angular2",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "description": "Official Angular 2 plugin for amCharts V3",
+  "module": "index.js",
+  "typings": "index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/amcharts/amcharts3-angular2.git"
   },
-  "peerDependencies": {
-    "@angular/core": "^2.0.0 || ^4.0.0"
```

Comment:
We [import `@angular/core` at runtime](https://github.com/amcharts/amcharts3-angular2/blob/e708364f412cc04af45f80cf9427b22fc112e78c/index.ts#L1) so it can't be in `devDependencies`

Putting it in `peerDependencies` is better than `dependencies` because it lets the user choose which version of `@angular/core` to use, and it ensures that there is only a single version of `@angular/core`. This is the recommended way to handle plugins with npm.